### PR TITLE
Instructions on setting up passwordless SSH

### PR DESCRIPTION
**Pull request by suhailrehman**
Added instructions on setting up passwordless SSH

### CODE REVIEW

File: environment/ssh.rst [hunk 1]
```diff
@@ -71,89 +78,248 @@ this::
...
 
-Once you've installed SSH, you should re-try the instructions at the top of the page.
+Once you've installed SSH, you should re-try the instructions at the top of the
+page.
+
+
+.. _passwordless-ssh:
+
+Setting up Passwordless SSH
+===========================
```
Comment (borjasotomayor) [inline environment/ssh.rst]: This heading needs to use a lower heading level, i.e.,

```
Setting up Passwordless SSH
---------------------------
```

If the intent is for this section to appear linked directly under "Software Environment", then it should be placed in a separate page.

File: environment/ssh.rst [hunk 2]
```diff
@@ -71,89 +78,248 @@ this::
...
+
+These instructions will allow you to configure your computer to connect to the
+Linux server without being prompted for your password each time, which is much
+more convenient. 
+
+You only need to follow these instructions once on your personal
+computer/laptop.
+
+.. warning :: 
+  If you have already completed the GitHub SSH setup instructions, you
```
Comment (borjasotomayor) [inline environment/ssh.rst]: Add "(from the GitHub Basics tutorial)" after "setup instructions" (with a link to the GitHub Basics tutorial)

File: environment/ssh.rst [hunk 3]
```diff
@@ -71,89 +78,248 @@ this::
...
+    Replace ``CNETID`` with your CNetID, and ``<servername>`` with the name of
+    the server you want to connect to. For example, if you want to connect to
+    such as ``linux1.cs.uchicago.edu`` or ``cs141-4.cs.uchicago.edu``.
+
+    You will be prompted for your password. Enter your CNetID password.
+
+- **Windows**: Run the following command:
+
+  ``$ type $env:USERPROFILE\\.ssh\\id_rsa.pub | ssh
+  CNETID@<servername>.cs.uchicago.edu “cat >> .ssh/authorized_keys”``
```
Comment (borjasotomayor) [inline environment/ssh.rst]: Use a code block (with three back-ticks) to ensure this long command is formatted better.

**Review by borjasotomayor — CHANGES_REQUESTED** [review]
Looks pretty good. Just a few minor things that need to be updated.

**Comment by suhailrehman** [comment]
I made the requested fixes, and also removed the `$` character from the commands to match how the instructions look like in other sections of the guide.

**Review by borjasotomayor — APPROVED** [review]
LGTM! :shipit:

**Comment by borjasotomayor** [comment]
These changes have been deployed to https://uchicago-cs.github.io/student-resource-guide/

**Pull request merged**